website/docs/sql-reference/statements/sql-group-by.md

```diff
@@ -0,0 +1,73 @@
+---
+id: group-by
+title: SQL GROUP BY
+description: The GROUP BY statement allows you to group query results by specified columns and is used in pair with aggregate functions such as AVG and SUM to calculate those values across specific rows.
+slug: /sql-reference/group-by
+---
+
+<head>
+    <title>Working with the SQL GROUP BY statement</title>
+</head>
+
+GROUP BY…it’s a little hard to explicitly define in a way *that actually makes sense*, but it will inevitably show up countless times in analytics work and you’ll need it frequently.
+
+To put it in the simplest terms, the GROUP BY statement allows you to group query results by specified columns and is used in pair with aggregate functions such as AVG and SUM to calculate those values across specific rows.
+
+## How to use the SQL GROUP BY statement
+
+The GROUP BY statement appears at the end of a query, after any joins and [WHERE](/sql-reference/where) filters have been applied:
+
+```sql
+select 
+	my_first_field,
+	count(id) as cnt --or any other aggregate function (sum, avg, etc.) 
+from my_table
+where my_first_field is not null
+group by 1 --grouped by my_first_field
+order by 1 desc
+```
+
+A few things to note about the GROUP BY implementation:
+- It’s usually listed as one of the last rows in a query, after any joins or where statements; typically you’ll only see [HAVING](/sql-reference/having), [ORDER BY](/sql-reference/order-by), or [LIMIT](/sql-reference/limit) statements following it in a query
+- You can group by multiple fields (ex. `group by 1,2,3`) if you need to; in general, we recommend performing aggregations and joins in separate CTEs to avoid having to group by too many fields in one query or CTE 
```

Comment:
```suggestion
- You can group by multiple fields (ex. `group by 1,2,3`) if you need to; in general, we recommend performing aggregations and joins in separate <Term id="cte">CTEs</Term> to avoid having to group by too many fields in one query or CTE 
```